pkg/tfvars/openstack/rhcos_image.go

```diff
@@ -47,10 +47,14 @@ func uploadBaseImage(cloud string, localFilePath string, imageName string, clust
 	}
 	logrus.Debugf("Image %s was created.", img.Name)
 
-	useImageImport, err := isImageImportSupported(cloud)
-	if err != nil {
-		return err
-	}
+	// FIXME(mfedosin): We have to temporary disable image import, because it looks
+	// like there are problems on the server side.
+	// Revert this patch when the problems are fixed.
+	useImageImport := false
+	// useImageImport, err := isImageImportSupported(cloud)
+	// if err != nil {
+	// 	return err
+	// }
```

Comment:
i do not think we should be tracking this in code as comments. I think we should have opened a 
- github issue or bz detailing the issue
- switch to import false with comment linking to the issue.

@Fedosin

Reply:
@abhinavdahiya You're absolutely right. Sorry I was pushing for this to merge ASAP as it was preventing us from running the CI on an alternate cloud provider.

I'll submit a patch that links to https://github.com/openshift/okd/issues/122